From 26e2ae53c67b8fcd33f3e772429f44167183b471 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 25 Jun 2020 06:44:17 +0300
Subject: [PATCH] [SW-14252] hl: boot-fit: add target property in uboot

As preboot needs a unique identifier for each image to perform actions,
target property is added for images available in preboot configuration.

Change-Id: I9e136e543ecbda6c469c7286e1adb669f5e57ebf
---

diff --git a/board/habanalabs/common/uboot.its b/board/habanalabs/common/uboot.its
index c5f0f12..8228d6a 100644
--- a/board/habanalabs/common/uboot.its
+++ b/board/habanalabs/common/uboot.its
@@ -92,6 +92,7 @@
 			description = "U-Boot (64-bit)";
 			data = /incbin/("@HL_DEV@-u-boot.bin.le");
 			type = "standalone";
+			target = "arm64";
 			arch = "arm64";
 			compression = "none";
 			loadaddr = <0x7F 0xF0100000>;
